src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift

```diff
@@ -1367,7 +1367,11 @@ enum AppStrings {
 
 	enum Statistics {
 		static let error = NSLocalizedString("Statistics_LoadingError", comment: "")
-		
+		enum AddCard {
+			static let SevenDayIncidence = NSLocalizedString("Statistics_Add_SevenDayIncidence", comment: "")
+			static let fromTheWholeCountry = NSLocalizedString("Statistics_Add_fromTheWholeCountry", comment: "")
```

Comment:
Just wondering, what is the difference between "whole country" and "nationwide"? 🤔

Reply:
They are actually different texts in German in the Design
the first is translated in german as  "**Gesamtes Bundesland**" , the other is "**Bundesweit**"

Reply:
Aaahh. I would think suggest something like 'whole state' then because 'country' is more the whole of Germany than just a… state/Bundesland.